atc/worker/placement.go

```diff
@@ -14,6 +14,7 @@ import (
 
 type PlacementOptions struct {
 	Strategies                   []string `long:"container-placement-strategy" default:"volume-locality" choice:"volume-locality" choice:"random" choice:"fewest-build-containers" choice:"limit-active-tasks" choice:"limit-active-containers" choice:"limit-active-volumes" description:"Method by which a worker is selected during container placement. If multiple methods are specified, they will be applied in order. Random strategy should only be used alone."`
+	NoInputStrategies            []string `long:"no-input-container-placement-strategy" default:"fewest-build-containers" choice:"volume-locality" choice:"random" choice:"fewest-build-containers" choice:"limit-active-tasks" choice:"limit-active-containers" choice:"limit-active-volumes" description:"Method by which a worker is selected during container placement. If multiple methods are specified, they will be applied in order. Random strategy should only be used alone."`
```

Comment:
```suggestion
	NoInputStrategies            []string `long:"no-input-container-placement-strategy" default:"fewest-build-containers" choice:"volume-locality" choice:"random" choice:"fewest-build-containers" choice:"limit-active-tasks" choice:"limit-active-containers" choice:"limit-active-volumes" description:"A second container placement strategy that will only be used for get and nested check steps."`
```

With my suggestion above having it so that if we don't pass anything in to this flag, it will just use the original strategy we will need to remove the default here then so that it can be a nil value if it is not passed in.